aqua-ethos-setup/aqua_ethos.sh

```diff
@@ -132,6 +244,26 @@ else
 	makePost "adminrules" '{"name":"core-user-rule","description": "Core User is Admin of all containers","role":"administrator","resources":{"containers":["*"],"images":["*"],"volumes":["*"],"networks":["*"]},"accessors":{"users":["core"]}}'
 fi
 
+# See if image assurance exists
+IMAGE_ASSURANCE=$(makeGet image_policy)
+
+if [[ "$IMAGE_ASSURANCE" == "200" ]]; then
+	log "image assurance already exists..."
+else
+	makePost "image_policy" '{"only_registered_images":true,"daily_scan_enabled":true,"daily_scan_time":{"hour":1,"minute":0,"second":0},"average_score_enabled":false,"average_score":7.5,"maximum_score_enabled":false,"maximum_score":8,"author":"system","cves_black_list_enabled":true,"cves_black_list":["CVE-2014-6271","CVE-2014-0160","CVE-2012-1723","CVE-2013-2465","CVE-2016-2842","CVE-2016-2108","CVE-2016-0705"],"allowed_images":[]}'
+fi
+
+# See if aqua qualys integration already exists
+EXISTING_QUALYS=$(makeGet settings/integrations/qualys)
+
+PROFILE_BODY_QUALYS="{\"enabled\": \"true\", \"url\": $QUALYS_URL, \"username\": $QUALYS_USERNAME, \"password\": $QUALYS_PASSWORD}"
```

Comment:
This is not valid JSON. The url, username, and password, need quotes around them unless they are numbers or booleans